<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IGIDataSourcePlaybackWidget</class>
 <widget class="QWidget" name="IGIDataSourcePlaybackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkIGIDataSourcePlayBackManager</string>
  </property>
  <property name="toolTip">
   <string>Press to advance time automatically.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ctkPathLineEdit" name="m_DirectoryChooser" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_PlayPushButton">
       <property name="toolTip">
        <string>play back data from a previously recorded session</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_RecordPushButton">
       <property name="toolTip">
        <string>record (save) data</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="m_FixedRecordTimeInterval">
       <property name="toolTip">
        <string>Set fixed record time in hours:mins:secs:millisecs or monitor the elapsed time.</string>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss:zzz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_StopPushButton">
       <property name="toolTip">
        <string>stop recording or playing</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_GrabScreenCheckbox">
       <property name="text">
        <string>grab screen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="m_ToolManagerPlaybackGroupBox">
     <property name="title">
      <string>Playback controls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="m_StartPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_PlayingPushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>|&gt;</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_EndPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_TimeStampEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="m_PlaybackSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
